extensions/replication/ReplicationAPI.js

```diff
@@ -93,6 +93,7 @@ class ReplicationAPI {
                 requestId: reqId,
                 // TODO: BB-217 do not use contentLength from metrics
                 size: contentLength,
+                etag: eTag,
```

Comment:
nit: should `t` be capitalized `T`?

Reply:
No, https://github.com/scality/sorbet/pull/31/files#diff-8f3c4cb253eee09ae2401daa7279a8bbfbfd4168bb579c3ac0ee5c672d63bb2cR137